src/util/init.js

```diff
@@ -687,6 +687,153 @@
     },
 
     /**
+     * Escape any character in a string that has a code point >= 127.
+     *
+     * @param {string} text: the string to escape.
+     * @returns {string}: the escaped string.
+     */
+    escapeUnicodeHTML: function (text) {
```

Comment:
The `&copy;` (&copy;) symbol in the attribution.  Generally, the SVG foreignObject seems to handle this as `&#169;`, but not as unescaped unicode.